src/Tidy/Codegen/Monad.purs

```diff
@@ -91,11 +104,35 @@ data CodegenImport
 derive instance Eq CodegenImport
 derive instance Ord CodegenImport
 
+-- | `import Foo` = `OpenHiding Set.empty`
+-- | `import Foo hiding (bar)` = `OpenHiding (Set.singleton bar)`
+-- | `import Foo (bar)` = `ClosedImporting (NES.singleton bar)`
+-- |
+-- | If a module is imported multiple times unqualified,
+-- | import forms on the left will be overridden by import forms on the right
+-- | ```
+-- | ClosedImport _ < OpenImport Set.empty < OpenImport nonEmptySet
+-- | ```
```

Comment:
Same comment regarding these docs. Who is the audience for this? What is "left" and "right"? Where are ClosedImport and OpenImport coming from, and how do they relate to ClosedImporting and OpenHiding. I'm just not sure what I'm supposed to get out of these doc comments as a downstream user.

Reply:
It's for lib maintainers. I was trying to say the below more succinctly.
Given the below two imports...
```purescript
import Foo (bar, baz)
import Foo
```
this will output
```purescript
import Foo
```
Given the below two imports
```purescript
import Foo
import Foo hiding (bar)
```
this will output
```purescript
import Foo hiding (bar)
```
Given the below imports
```purescript
import Foo (bar, baz)
import Foo
import Foo hiding (bar)
```
this will output
```purescript
import Foo hiding (bar)
```

Reply:
I've renamed `ClosedImporting` to just `Closed`